src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -97,13 +128,21 @@ export const DeeplinkContextProvider: React.FC<IDeeplinkContextProviderProps> =
         sort: false
     }) as unknown) as IPublicDeeplink;
 
+    const selectedAdtInstanceInLocalStorage = getSelectedAdtInstanceFromLocalStorage();
+    const selectedStorageContainerInLocalStorage = getSelectedStorageContainerFromLocalStorage();
```

Comment:
since these get... ones doing the migration as well, no matter we use it or not in a single spot I need to call them. Also since they are in or condition with || they are needed to be evaluated no matter what.